File: app/src/main/java/com/neome/java/api/home/drawer/msg/MsgEntFilter.java
// neome.ai API - do not change
//

package com.neome.java.api.home.drawer.msg;

import com.neome.java.api.core.base.msg.MsgVersion;
import com.neome.java.api.meta.base.Types.EntId;

import org.jetbrains.annotations.Nullable;

import java.util.Set;

@SuppressWarnings("unused")
public class MsgEntFilter extends MsgVersion
{
  @Nullable
  public Set<EntId> filterEntIdSet;
}

File: app/src/main/java/com/neome/api/home/drawer/msg/MsgEntFilter.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.drawer.msg

import com.neome.api.meta.base.Types.EntId
import com.neome.api.core.base.msg.MsgVersion

interface MsgEntFilter : MsgVersion
{
  val filterEntIdSet: Set<EntId>?
}